Makefile

```diff
@@ -146,6 +146,11 @@ cluster-sync-handler: cluster-sync-resources $(local_handler_manifest)
 	$(KUBECTL) apply -f deploy/crds/nmstate_v1alpha1_nodenetworkconfigurationpolicy_crd.yaml
 	$(KUBECTL) delete --ignore-not-found -f $(local_handler_manifest)
 	$(KUBECTL) create -f $(local_handler_manifest)
+	until [[ \
```

Comment:
Can't we use kubectl wait here ? 

Also desiredNumberScheduler can be retrieved out of the loop

Reply:
We can't, `wait` can be used only for conditions. The DS readiness is not exposed in any condition (maybe if we digged into replica-set...).

We can retrieve it outside the loop, but we cannot store any variables in make target, can we.

Reply:
Ack about wait :-/

About variable, let's use this PR to move bash code of cluster-sync to hack/cluster-sync.sh, let's export KUBECTL as an environment variable.